ext/mysqli/mysqli.stub.php

```diff
@@ -0,0 +1,773 @@
+
+	/**
+	 * @return bool
+	 */
+    public function store_result();
+
+	/**
+	 * @return mysqli_result|bool
+	 */
+    public function get_result();
+}
+
+class mysqli_sql_exception extends RuntimeException
+{
+}
+
+function mysqli_affected_rows(mysqli $mysql_link): int|bool {}
+
+function mysqli_autocommit(mysqli $mysql_link, bool $mode): bool {}
+
+function mysqli_begin_transaction(mysqli $mysql_link, int $flags = 0, string $name = ''): bool {}
+
+function mysqli_change_user(mysqli $mysql_link, string $user, string $password, ?string $database): bool {}
+
+function mysqli_character_set_name(mysqli $mysql_link): string|bool {}
+
+function mysqli_close(mysqli $mysql_link): bool {}
+
+function mysqli_commit(mysqli $mysql_link, int $flags = 0, string $name = ''): bool {}
+
+function mysqli_connect(?string $host = null, ?string $user = null, ?string $password = null, ?string $database = null, ?int $port = null, ?string $socket = null): mysqli|bool {}
+
+function mysqli_connect_errno(): int {}
+
+function mysqli_connect_error(): ?string {}
+
+function mysqli_data_seek(mysqli_result $mysql_result, int $offset): bool {}
+
+function mysqli_dump_debug_info(mysqli $mysql_link): bool {}
+
+function mysqli_debug(string $debug): bool {}
+
+function mysqli_errno(mysqli $mysql_link): int|bool {}
+
+function mysqli_error(mysqli $mysql_link): string|bool {}
+
+function mysqli_error_list(mysqli $mysql_link): array {}
+
+function mysqli_stmt_execute(mysqli_stmt $mysql_stmt): bool {}
+
+function mysqli_execute(mysqli_stmt $mysql_stmt): bool {}
+
+function mysqli_fetch_field(mysqli_result $mysql_result): object|false {}
+
+function mysqli_fetch_fields(mysqli_result $mysql_result): array|bool {}
+
+function mysqli_fetch_field_direct(mysqli_result $mysql_result, int $offset): object|false {}
+
+function mysqli_fetch_lengths(mysqli_result $mysql_result): array|false {}
+
+function mysqli_fetch_all(mysqli_result $mysql_result, int $mode = MYSQLI_NUM): array|false {}
+
+function mysqli_fetch_array(mysqli_result $mysql_result, int $fetchtype = MYSQLI_BOTH): array|false|null {}
+
+function mysqli_fetch_assoc(mysqli_result $mysql_result): array|bool|null {}
+
+function mysqli_fetch_object(mysqli_result $mysqli_result, string $class_name = 'stdClass', array $params = []): object|bool|null {}
+
+function mysqli_fetch_row(mysqli_result $mysqli_result): array|bool|null {}
+
+function mysqli_field_count(mysqli $mysqli_link): int|bool {}
+
+function mysqli_field_seek(mysqli_result $mysqli_result, int $field_nr): bool {}
+
+function mysqli_field_tell(mysqli_result $mysqli_result): int|bool {}
+
+/**
+ * @param mysqli_result $mysqli_result
+ *
+ * @return void
+ */
+function mysqli_free_result(mysqli_result $mysqli_result) {}
+
+function mysqli_get_connection_stats(mysqli $mysqli_link): array {}
+
+function mysqli_get_client_stats(): array {}
+
+function mysqli_get_charset(mysqli $mysqli_link): object|bool {}
+
+function mysqli_get_client_info(): string {}
```

Comment:
As far as I can see, it can return `null` as well because: 
```C
const char * info = mysql_get_client_info();
if (info) {
	RETURN_STRING(info);
}
```

If @cmb69 confirms this, please extend the following line in opcache:
`F1("mysqli_get_client_info",				MAY_BE_STRING),` with ` | MAY_BE_NULL`